docs/make.jl

```diff
@@ -47,6 +51,7 @@ pages = [
         "Single-column ocean simulation" => "literated/single_column_os_papa_simulation.md",
         "One-degree ocean--sea ice simulation" => "literated/one_degree_simulation.md",
         "Near-global ocean simulation" => "literated/near_global_ocean_simulation.md",
+        #"Panantarctic regional simulation" => "literated/panantarctic_regional_simulation.md",
```

Comment:
```suggestion
        "Panantarctic regional simulation" => "literated/panantarctic_regional_simulation.md",
```

uncomment this to include the example in the docs

Reply:
Thanks!  Maybe I'll keep this commented for the moment, to make sure that everything passes without this example.  If yes, then I'll add the exampe back in.